clintfidel/WEconnect
# Ft testing branch

### What does this PR do?
This PR ensures that all unit, integration and end-to-end testing are all in place 

#### Description of Task to be completed?
- write test for action, reducers and components
- write end-to-end test
#### How should this be manually tested?
- clone the repository
- npm install
- enter this command -npm run client:test for integration test
- npm run e2e:test for end-to-end test, make sure to do npm run start-server:e2e
